### PR TITLE
SS-1229 Fix of the bug not saving deleted on date

### PR DESCRIPTION
**Pull request by akochari**
## Description

This PR fixes the issue SS-1229. We didn't save the _deleted_on_ date when an app was deleted by the users. In addition, I added timezone info to the saved value. I also added a unit test to check that this field was populated on deletion.
This PR also fixes the issue SS-1238. Persistent volumes that were deleted were shown as options when launching a file manager or notebooks.
Finally, unrelated change - fixed the links to documentation from the app creation/edit form because we missed that in past PRs.

## Checklist

_If you're unsure about any of the items below, don't hesitate to ask. We're here to help!
This is simply a reminder of what we are going to look for before merging your code._

- [x] This pull request is against **develop** branch (not applicable for hotfixes)
- [x] I have included a link to the issue on GitHub or JIRA (if any)
- [ ] I have included migration files (if there are changes to the model classes)
- [x] I have included, reviewed and executed tests (unit and end2end) to complement my changes
- [ ] I have updated the related documentation (if necessary)
- [x] I have added a reviewer for this pull request
- [x] I have added myself as an author for this pull request
- [ ] In the case I have modified settings.py, then I have also updated the studio-settings-configmap.yaml file in serve-charts

### CODE REVIEW

File: studio/settings.py [hunk 1]
```diff
@@ -38,6 +38,9 @@
 # Since this file is only used for development, we can have this set to all hosts.
 ALLOWED_HOSTS = ["*"]
 
+# Timezone settings
+USE_TZ = True
+TIME_ZONE = "Europe/Stockholm"
```
Comment (alfredeen) [inline studio/settings.py:43]: Line 295 also contains the TIME_ZONE setting and should be removed.

```suggestion
Please remove line 295 with the TIME_ZONE setting.
```

File: studio/settings.py
Comment (alfredeen) [inline studio/settings.py:1]: Line 295 also contains the TIME_ZONE setting and should be removed.

**Review by alfredeen — CHANGES_REQUESTED** [review]

File: studio/settings.py
Comment (akochari) [reply studio/settings.py:1]: I thought we could set the timezone. Is that not a good idea?

Comment (alfredeen) [reply studio/settings.py:1]: Yes it is a good idea. The problem is that there is an existing line with TIME_ZONE = "UTC" that needs to be deleted so we do not have duplicate rows.

**Review by alfredeen — APPROVED** [review]
LGTM. Good work.

**Pull request merged**